src/main/java/com/archyx/aureliumskills/api/event/ManaAbilityReadyEvent.java

```diff
@@ -0,0 +1,39 @@
+package com.archyx.aureliumskills.api.event;
+
+import com.archyx.aureliumskills.mana.MAbility;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+public class ManaAbilityReadyEvent extends Event {
```

Comment:
I think the name should be changed to ManaAbilityRefreshEvent to not get confused with the readying when you right-click the tool before activating the ability.